default/python/chat/chat.py

```diff
@@ -0,0 +1,42 @@
+from logging import info
+
+from common.configure_logging import redirect_logging_to_freeorion_logger
+
+# Logging is redirected before other imports so that import errors appear in log files.
+redirect_logging_to_freeorion_logger()
+
+
+class ChatProvider:
```

Comment:
What exactly is this providing, or will it provide in general? Just chat history? If so, maybe name it more specifically?

Reply:
Will `ChatHistoryProvider` be better name?

Reply:
Changed.